pom.xml

```diff
@@ -35,7 +35,8 @@
     <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
     <jetty.version>9.3.8.v20160314</jetty.version>
     <appengine.api.version>1.9.40</appengine.api.version>
-    <docker.label.prefix>async-</docker.label.prefix>
+    <docker.tag>${jetty.version}</docker.tag>
```

Comment:
Then I'd recommend making the artifact version the build ID.  we then get auto increment whenever a release is done - no matter where that is released to.